blog/2025-08-26-verifying-bls12-on-ethereum.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Verifying the `quicknet` beacons on Ethereum"
+description: "At long last!"
+date: 2025-08-26
+author: Julie Bettens
+tags:
+    - Beginner
+---
+Let us explore together the effectivness of EIP-2537 after the Pectra upgrade for onchain BLS signature verification.
+<!-- truncate -->
+
+The League of Entropy has been emitting `quicknet` random beacons for anyone to use for all their public verifiable randomness uses [since 2023](https://docs.drand.love/blog/2023/10/16/quicknet-is-live), and ever since we launched it people have been repeatedly asking us about how to verify drand beacons “onchain”, especially on Ethereum. Today we finally have everything we need to answer that question!
```

Comment:
```suggestion
The League of Entropy has been emitting `quicknet` random beacons for anyone to use for all their public verifiable randomness needs [since 2023](https://docs.drand.love/blog/2023/10/16/quicknet-is-live), and ever since we launched it people have been repeatedly asking us about how to verify drand beacons “onchain”, especially on Ethereum. Today we finally have everything we need to answer that question!
```